src/order/category/LinearOrder.lean

```diff
@@ -0,0 +1,25 @@
+/-
+Copyleft 2020 Johan Commelin. No rights reserved.
+Authors: Johan Commelin
```

Comment:
If I recall the discussion correctly, the consensus was to stick with the Apache headers in mathlib for now.

Reply:
Oops, fixed